.github/workflows/atlas-tests.yml

```diff
@@ -0,0 +1,134 @@
+## Github workflow to run atlas tests
```

Comment:
Yes, making these tests separate makes a lot of sense to me! Especially since they're so slow.

What do you think of adding more of a description to either the top of this file or the `Run Tests` job on line 101 that'll justify the history of these tests. Something like 

```yaml
## Github workflow for atlas tests. 
## Historically these tests take far longer than the others in the CI so we've separated them out
## For speed of testing.
```

Reply:
added some top-level file comments

Reply:
> Yes, making these tests separate makes a lot of sense to me! Especially since they're so slow.
> 
> What do you think of adding more of a description to either the top of this file or the `Run Tests` job on line 101 that'll justify the history of these tests. Something like
> 
> ```yaml
> ## Github workflow for atlas tests. 
> ## Historically these tests take far longer than the others in the CI so we've separated them out
> ## For speed of testing.
> ```

I'm not sure there will be a benefit to adding comments in the `Run Tests` step - the implication of separating the tests from a single workflow is we can now parallelize *more* tests. 
I went for a logical separation based on what type of test is running (i.e. atlast, epoch, etc). 

i've added some top-level comments so it should be more clear what a `*-test.yml` yaml is doing, but i don't think there would be a benefit of adding more comments to the step itself.